src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/utils/CaseDataBuilder.java

```diff
@@ -135,7 +136,7 @@ private HearingTypeItem createHearing(String hearingNumber, String hearingType,
 
     public CaseDataBuilder withHearingSession(int hearingIndex, String number, String listedDate, String hearingStatus,
                                               boolean disposed) {
-        var hearing = caseData.getHearingCollection().get(hearingIndex);
+        final var hearing = caseData.getHearingCollection().get(hearingIndex);
 
         var dateListedType = new DateListedType();
```

Comment:
Where you're changing something that's declared with var, please clean it up

Reply:
I don't quite understand. To clarify my change, this code wasn't mine, I only wanted to improve the checkstyle so I made it constant. I've changed it's position now and made it a type 'val' to remove the checkstyle problem.